pkgs/development/libraries/science/math/caffe2/update_clang_cvtsh_bugfix.patch

```diff
@@ -0,0 +1,55 @@
+diff --git a/caffe2/perfkernels/cvtsh_ss_bugfix.h b/caffe2/perfkernels/cvtsh_ss_bugfix.h
```

Comment:
Is there an external source for this patch? It is generally preferred to fetch patches (using e.g., `fetchpatch`).

Reply:
Thanks for the suggestion. As a single patch, I suspect not.

557f877 in https://github.com/caffe2/caffe2/commits/master/caffe2/perfkernels/cvtsh_ss_bugfix.h is in v0.8.1 and there're 4-5 patches upstream to cover various versions/flavors of clang.

But then, what the patch does is to replace `cvtsh_ss_bugfix.h` at v0.8.1 by that of the current master, so I could fetch the latest `cvtsh_ss_bugfix.h` and override by it in one of the build hooks.